src/main/java/net/imagej/ops/deconvolve/RichardsonLucyC.java

```diff
@@ -0,0 +1,184 @@
+ * %%
+ * Copyright (C) 2014 - 2016 Board of Regents of the University of
+ * Wisconsin-Madison, University of Konstanz and Brian Northan.
+ * %%
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ * 
+ * 1. Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ * 
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDERS OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ * #L%
+ */
+
+package net.imagej.ops.deconvolve;
+
+import net.imagej.ops.Ops;
+import net.imagej.ops.filter.AbstractIterativeFFTFilterC;
+import net.imagej.ops.special.computer.AbstractUnaryComputerOp;
+import net.imagej.ops.special.computer.BinaryComputerOp;
+import net.imagej.ops.special.computer.Computers;
+import net.imglib2.Cursor;
+import net.imglib2.RandomAccessibleInterval;
+import net.imglib2.img.Img;
+import net.imglib2.outofbounds.OutOfBoundsConstantValueFactory;
+import net.imglib2.outofbounds.OutOfBoundsFactory;
+import net.imglib2.type.Type;
+import net.imglib2.type.numeric.ComplexType;
+import net.imglib2.type.numeric.RealType;
+import net.imglib2.util.Util;
+import net.imglib2.view.Views;
+
+import org.scijava.Priority;
+import org.scijava.app.StatusService;
+import org.scijava.plugin.Parameter;
+import org.scijava.plugin.Plugin;
+
+/**
+ * Richardson Lucy algorithm for (@link RandomAccessibleInterval) (Lucy, L. B.
+ * (1974).
+ * "An iterative technique for the rectification of observed distributions".)
+ * 
+ * @author Brian Northan
+ * @param <I>
+ * @param <O>
+ * @param <K>
+ * @param <C>
+ */
+
+@Plugin(type = Ops.Deconvolve.RichardsonLucy.class,
+	priority = Priority.HIGH_PRIORITY)
+public class RichardsonLucyC<I extends RealType<I>, O extends RealType<O>, K extends RealType<K>, C extends ComplexType<C>>
+	extends AbstractIterativeFFTFilterC<I, O, K, C> implements
+	Ops.Deconvolve.RichardsonLucy
+{
+
+	@Parameter(required = false)
```

Comment:
This is because in the unit tests, the `StatusService` is not loaded to the `context`. Remove the `(required = false)` and add these lines to `DeconvolveTest` and any places that would use `RichardsonLucyC`:
```
@Override
public Context createContext() {
	return new Context(OpService.class, OpMatchingService.class,
		CacheService.class, StatusService.class);
}
```
Check if this works.